test/test_model.py

```diff
@@ -61,6 +61,29 @@ class TestModelFunctionality(unittest.TestCase):
     def setUp(self):
         pass
 
+    def __helper_single_letter_species_names(self,model,x):
+        try:
+            model.add_species(spatialpy.Species(x,0))
+            print("checking '"+x+"'")
+        except Exception as e:
+            raise e
+
+    def test_single_letter_species_names(self):
+        """ Test that we can have any single letter species name"""
+        model = spatialpy.Model()
+        reserved_names = ["x", "vol","sd","data_fn","t","debug_flag","Spatialpy"]
+        for x in string.ascii_letters:
+            if x in reserved_names:
+                self.assertRaises(spatialpy.ModelError, lambda:self.__helper_single_letter_species_names(model,x) )
+            else:
+                model.add_species(spatialpy.Species(x,0))
+        model.set_timesteps(output_interval=1,num_steps=1,timestep_size=1)
+        model.set_domain(spatialpy.Domain.create_2D_domain([0,1],[0,1],2,2))
```

Comment:
Lines 80 and 81 should be in separate tests

Reply:
To do a model.run() you need to have a time span and a domain, thus both of these are required.